public/style.css

```diff
@@ -269,6 +293,17 @@ form div {
   border-radius: 0.5rem;
   cursor: pointer;
 }
+@media (max-width: 700px) {
+  .addToCartBtnDetail {
+    width: 40%;
+    height: 2rem;
+    font-size: 1.2rem;
+    background-color: green;
+    color: white;
+    border-radius: 0.5rem;
+    cursor: pointer;
+  }
+}
```

Comment:
for some reason I don't see the part working... let me know if I'm wrong

Reply:
I just added some functionality in the css to fix what I think you're referring to. I can demo it after stand up before I merge it in